dsbapi/__init__.py

```diff
@@ -19,7 +19,6 @@
 except:
     import Image
 import pytesseract
-import requests
```

Comment:
this breaks it, requests is needed.

Reply:
![DSBApi-screenshot](https://user-images.githubusercontent.com/102413837/166495586-3b77a269-5c40-4c54-a145-a9b666bc4bf4.jpg)